src/components/Content.tsx

```diff
@@ -88,6 +88,7 @@ const Content: FC = () => {
         id='content'
         ref={contentRef}
         className={contentClassNames}
+        style={{ marginTop: rootThoughtsLength > 0 ? '0.501em' : '0px' }}
```

Comment:
Let's put this in [LayoutTree](https://github.com/cybersemics/em/blob/ac0f5c773f1b6e3312baa9b000d563aa09b2db59/src/components/LayoutTree.tsx) so that we don't need to have the conditional.